clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -595,6 +595,7 @@ Expected<StringRef> linkDevice(ArrayRef<StringRef> InputFiles,
   case Triple::aarch64_be:
   case Triple::ppc64:
   case Triple::ppc64le:
+  case Triple::spirv64:
```

Comment:
Some recent changes here for `looongarch` might need to rebase.

Reply:
Will do, thanks